Documentation/ApiOverview/Events/Events/Backend/_IsContentUsedOnPageLayoutEvent/_ContentUsedOnPage.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MyVendor\MyExtension\Listener;
+
+use TYPO3\CMS\Backend\View\Event\IsContentUsedOnPageLayoutEvent;
+use TYPO3\CMS\Core\Attribute\AsEventListener;
+
+#[AsEventListener(
```

Comment:
The `AsEventListener` attribute is a new feature for main (13.0).

Using the attribute AND the registration over `Services.yaml` is duplicated.

Not sure about the policy for the documentation, but:

* remove the `Services.yaml` for main and for the backport use the `Services.yaml` and drop the PHP attribute usage
* or simply use only the `Services.yaml` version